### ISSUE TITLE
Appears the UI is broken

### ISSUE THREAD

**Issue by wd5gnr** [seancolyer/gmail-crypt#83]
No longer able to see buttons in recent Gmail. If you look at the Extension store, lots of people having the same issue. No workaround that I know of.

**Comment by ChicagoGregg** [comment]
I agree. The latest as of this writing is that the UI isn't appearing in Gmail. Any fix to this?

**Comment by Ruisseaux** [comment]
Yes, as I think your tool is very useful, please update when you have time...
